xml/dm_registers.xml

```diff
@@ -103,27 +105,30 @@
         as well as the currently selected harts, as defined in \Fhasel.
 
         <!-- Fields that apply to a specific hart. -->
-        <field name="haltreq" bits="31" access="R/W" reset="0">
-            Halt request signal for all currently selected harts. When set to
-            1, each selected hart will halt if it is not currently halted.
+        <field name="haltreq" bits="31" access="W" reset="-">
+            Writes the halt request bit for all currently selected harts.
+            When set to 1, each selected hart will halt if it is not currently
+            halted.
 
 	    Writing 1 or 0 has no effect on a hart which is already halted, but
 	    the bit should be cleared to 0 before the hart is resumed.
 
             Writes apply to the new value of \Fhartsel and \Fhasel.
         </field>
-        <field name="resumereq" bits="30" access="R/W" reset="0">
-            Resume request signal for all currently selected harts. When set to 1,
-            each selected hart will resume if it is currently halted.
+        <field name="resumereq" bits="30" access="W" reset="-">
+            Writes the resume request bit for all currently selected harts.
+            When set to 1, each selected hart will resume if it is currently
+            halted.
 
-            This bit is ignored while \Fhaltreq is set.
+            The resume request bit is ignored while the halt request bit is
```

Comment:
This is a backwards compatible change by the way (not from this actual PR but I think this was added recently), and could be hard to implement. It could conflict with possible implementations of quick access. Wasn't this undefined before?

Reply:
sorry I meant "incompatible"

Reply:
I don't understand how this affects anything. It's saying that the debugger must clear haltreq before (or at the same time as) settings resumereq. Does it sound like something different is meant?

Reply:
It says "the resume request is ignored while...". That is a stronger HW requirement than "Setting both resumereq and haltreq at the same time leads to undefined behavior".

Reply:
Good point. I think I like this better since it forces sane debugger behavior.